plugins/sqlfluff-templater-dbt/sqlfluff_templater_dbt/templater.py

```diff
@@ -381,7 +381,18 @@ def _find_node(self, fname, config=None):
 
         if not results:
             model_name = os.path.splitext(os.path.basename(fname))[0]
-            disabled_model = self.dbt_manifest.find_disabled_by_name(name=model_name)
+            if DBT_VERSION_TUPLE >= (1, 0):
```

Comment:
The function `find_disabled_by_name()` apparently no longer exists in dbt 1.0. I didn't find an equivalent function, so here I'm looping over all disabled models to find a match.